plotPattern.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python3
+
+"""
+Created on Wed May 24 09:44:53 2017
+
+Plot the pattern for New Chip Test Box (.pat)
+
+Changes:
+    - 2017-11-21 Adapt it to python-3
+    - 2017-09-25 All can be plotted
+    - 2017-09-22 Can be plotted but the loop and wait not work yet
+
+@author: zhang_j1
+"""
+import matplotlib.pyplot as plt
+from numpy import *
```

Comment:
Minor but could you change it to:

```python
import numpy as np
import matplotlib.pyplot as plt
```
and remove the * imports? With such large libraries as numpy and matplotlib it's hard to keep track of what you are importing

Reply:
done